changelog/13669.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
Should this be considered an improvement? It's exposing back the username as alias metadata, but doesn't change the `Alias.Name` itself (i.e. reverting back to the old behavior or anything like that).

Reply:
Yeah, good catch! This was a bug earlier because I reverted the change but we opt'd to keeping it as is and adding the username to `alias.metadata.name`